src/stores/chatExtras.js

```diff
@@ -25,11 +26,42 @@ import Vue from 'vue'
 
 import { getUserAbsence } from '../services/participantsService.js'
 
+/**
+ * @typedef {string} Token
+ */
+
+/**
+ * @typedef {object} State
+ * @property {{[key: Token]: object}} absence - The absence status per conversation.
+ * @property {{[key: Token]: number}} parentToReply - The parent message id to reply per conversation.
+ * @property {{[key: Token]: string}} chatInput -The input value per conversation.
```

Comment:
```suggestion
 * @property {Object.<Token, Object>} absence - The absence status per conversation.
 * @property {Object.<Token, number>} parentToReply - The parent message id to reply per conversation.
 * @property {Object.<Token, string>} chatInput -The input value per conversation.
```

There is no syntax in JSDoc but the documentation state to define it like that 
https://jsdoc.app/tags-type

Reply:
![image](https://github.com/nextcloud/spreed/assets/93392545/156355f9-01a6-46c4-b2b7-1e1b7ffd1003)

Anyway, it is here just for reference, until we get Typescript (one day)